compiler/src/Language/Bond/Codegen/Cpp/Grpc_h.hs

```diff
@@ -263,11 +263,11 @@ inline #{className}::#{proxyName}<TThreadPool>::#{proxyName}(
             Async#{methodName}(::std::make_shared< ::grpc::ClientContext>(), #{bonded methodInput}{request});
         }|]
 
-        privateProxyMethodDecl Function{..} = [lt|const ::grpc::RpcMethod rpcmethod_#{methodName}_;|]
-        privateProxyMethodDecl Event{..} = [lt|const ::grpc::RpcMethod rpcmethod_#{methodName}_;|]
+        privateProxyMethodDecl Function{..} = [lt|const ::grpc::internal::RpcMethod rpcmethod_#{methodName}_;|]
```

Comment:
Please run `$ stack test --test-arguments --accept` from the `compiler/tests` folder in order to update the `gbc` test files under `compiler/tests/generated`.

Reply:
Thanks, just did it, waiting for CI